docs/general/builtins/scalar.rst

```diff
@@ -2428,6 +2428,31 @@ Example:
     SELECT 1 row in set (... sec)
 
 
+.. _obj_description:
+
+``obj_description(integer, text)``
+----------------------------------
+
+This function exists mainly for compatibility with PostgreSQL. In PostgreSQL it
```

Comment:
would reword the second sentence to start "In PostgreSQL, the function returns..."